pxd/config.pxi.in

```diff
@@ -39,3 +39,4 @@ DEF HAVE_ZPOOL_LOAD_POLICY_T = @HAVE_ZPOOL_LOAD_POLICY_T@
 DEF HAVE_ZFS_ITER_SNAPSHOTS = @HAVE_ZFS_ITER_SNAPSHOTS@
 DEF HAVE_ZVOLSIZE_TO_RESERVATION_PARAMS = @HAVE_ZVOLSIZE_TO_RESERVATION_PARAMS@
 DEF HAVE_SENDFLAGS_T_VERBOSITY = @HAVE_SENDFLAGS_T_VERBOSITY@
+DEF HAVE_O3X = @HAVE_O3X@
```

Comment:
I think it's not a good approach to make this platform specific - we should perhaps be checking for existence of `thread_init` and `thread_fini` in the configure.ac and defining them here. As these are only macos specific, they won't exist on other platforms and not executed during libzfs usage

Reply:
This is now fixed in 9657ab4